api/tests/Feature/DepartmentTest.php

```diff
@@ -319,4 +323,85 @@ public function testCanViewPools()
         ]);
 
     }
+
+    // Exercise archiving a department and then reversing it
+    public function testArchivingAndUnArchiving()
+    {
+        $testDepartment = Department::factory()->create(['archived_at' => null]);
+        $user = User::factory()->asAdmin()->create();
```

Comment:
For some reason I was getting authorization issues using the `$this->adminUser` here so I just seeded another admin user 🤷

Reply:
Let me know if you want me to tinker with this, too.

Reply:
Somehow fixed 🤷 
Removed it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/13901/commits/18cc192af5603407cdb1a74b9df9f11f63930422